hutch_python/cli.py

```diff
@@ -98,17 +99,15 @@ def configure_ipython_session():
     # Set up tab completion modifications
     configure_tab_completion(ipy_config)
 
-    # remove force exit key bind from <ctrl-\\>
-    ipy_config.InteractiveShellApp.exec_lines = (
-        "import IPython; "
-        "from prompt_toolkit.keys import Keys; "
-        "ip = IPython.get_ipython(); "
-        "cb = Keys.ControlBackslash; "
-        "ip.pt_app.key_bindings.remove(cb) "
-    )
-
-    # add env info to ipython banner
-    ipy_config.TerminalInteractiveShell.banner2 = get_env_info()
+    # disable default banner
+    ipy_config.TerminalIPythonApp.display_banner = False
+
+    # Run startup hook code, print banner after startup hook files
+    files = [
+             str(Path(__file__).parent / 'startup_script.py'),
+             str(Path(__file__).parent / 'print_hint_banner.py')
+            ]
+    ipy_config.InteractiveShellApp.exec_files = files
```

Comment:
What happens if you do `hutch_python my_script.py`; does that go to `exec_files` or elsewhere?

Reply:
Good point.  The IPython shell wouldn't be initialized, so the entire `configure_ipython_session` block (and `exec_files`) would be ignored entirely.  

I suppose if someone's script requires the ELogPoster callback, the script wouldn't work

Reply:
It probably doesn't matter in the end, but I was curious if that was the same mechanism.

Reply:
fwiw I suspect this script option is under-utilized due to slow startup speed